js/languages/en-US.json

```diff
@@ -358,9 +356,17 @@
       "server": {
         "sectionName": "Server",
         "management": "Management",
+        "managementHelper": "Switch and manage server connections",
+        "managementBtn": "Connection Management",
         "peers": "Peers",
-        "btnShowManagementAction": "Connection Management",
-        "btnShowPeersAction": "Show Connected Peers"
+        "peersHelper": "Peers you're connected to",
+        "peersBtn": "Show Connected Peers",
+        "purge": "Purge Shared Files",
+        "purgeHelper": "Delete files you are sharing with others",
+        "purgeBtn": "Purge Files",
+        "purgeToolTip": "OpenBazaar shares content you have viewed with other users. If you have viewed text or images you don't want to share this button will delete them. This will not affect any files that belong to you.",
+        "purgeError": "There was an error, your cached files may not have been deleted.",
+        "purgeComplete": "Your cached files have been deleted."
```

Comment:
You may want to check with @cpacia to know if 'ob/purgecache' returns after the purge is complete or if it just kicks off a purge, returns immediately, and the purge finished at some time. That's how the publish api works. If the purge API works the same way, I think we'd want to tweak the messaging since `Your cached files have been deleted.` wouldn't be completely accurate.